integration-tests/src/tests/rename.test.ts

```diff
@@ -69,6 +69,7 @@ suite('Bitbake Rename Test Suite', () => {
           docUri,
           position
         )
+      // eslint-disable-next-line @typescript-eslint/no-unused-vars
       } catch (error) {
         // pass
```

Comment:
We don't care about the error, but `catch` is required to have an argument. `catch (_)` or `catch (_error)` was not accepted. I'd be willing to add such a rule if it can be found, since I agree this looks ridiculous.

Reply:
I was thinking about failing the test here with `fail()`.  But I am all good if the error is trivial.

Reply:
The behavior of assertWillComeTrue is to retry until the returned condition is true. So this works fine. If one command throws (VSCOde not initialized yet, ...), then the result will be false and the test not succeed untill an iteration works. "pass" here just means go along, not that the test has succeeded